src/app/domain/geolocation/hooks/tracking.ts

```diff
@@ -1,16 +1,38 @@
 import { useEffect } from 'react'
 
 import { useClient } from 'cozy-client'
+import Minilog from 'cozy-minilog'
 
 import {
   isGeolocationTrackingEnabled,
   checkShouldStartTracking
 } from '/app/domain/geolocation/services/tracking'
 import { checkGeolocationQuota } from '/app/domain/geolocation/helpers/quota'
+import {
+  GeolocationTrackingEmitter,
+  TRIP_END
+} from '/app/domain/geolocation/tracking/events'
+
+const log = Minilog('📍 Geolocation')
 
 export const useGeolocationTracking = (): void => {
   const client = useClient()
 
+  useEffect(() => {
+    if (!client) return
+
+    const onTripEnd = (): void => {
+      log.debug('Trip end event received, checking quota')
+      void checkGeolocationQuota(client)
```

Comment:
That's the first time I see an explicit `void` in a function call. Is it required by typescript?